domain_coordinator/cmake/run_test_isolated.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
This should be in a `script` folder (or maybe `bin`). `cmake` doesn't seem to be a good name for it.

Reply:
If we add it as an entry point, the script should be moved inside `domain_coordinator/domain_coordinator` (see comment below).

Reply:
I did it this way to match the way run_test.py was provided by [ament_cmake_test](https://github.com/ament/ament_cmake/tree/master/ament_cmake_test/cmake)

I don't know enough about **why** it was done that way to know if we should keep doing it that way, or if it's safe to do it a different way, but if you're willing to own that the change is safe, I'm willing to make the change.